filebeat-yml-script.py

```diff
@@ -120,6 +120,34 @@ def _get_additional_fields():
     return fields
 
 
+def _add_rename_fields():
+    yaml = YAML()
+    with open(FILEBEAT_CONF_PATH) as filebeat_yaml:
+        config_dic = yaml.load(filebeat_yaml)
+
+    fields = []
+    for entry in os.environ["renameFields"].split(";"):
+        fields.append(get_rename_field(entry, ","))
+
+    rename_fields = {"rename": {"fields": fields}}
+    config_dic["filebeat.inputs"][0]["processors"].append(rename_fields)
+
+    with open(FILEBEAT_CONF_PATH, "w+") as updated_filebeat_yml:
+        yaml.dump(config_dic, updated_filebeat_yml)
+
+
+def get_rename_field(entry, delimiter):
+    try:
+        old_key, new_key = entry.split(delimiter)
+    except ValueError:
+        raise ValueError("Failed to parse entry: {}".format(entry))
```

Comment:
get you change it to a more explainable error - something like 
Your "renameField" format  isn't correct please check in the documentation for the right format